api/views/volunteer.py

```diff
@@ -56,6 +58,46 @@ def summary(self, request: Request, username: str = None) -> Response:
         user = get_object_or_404(BlossomUser, username=username, is_volunteer=True)
         return Response(self.serializer_class(user).data)
 
+    @csrf_exempt
+    @swagger_auto_schema(
+        manual_parameters=[Parameter("username", "query", type="string")],
+        responses={
+            400: 'No "username" as a query parameter.',
+            404: "No volunteer with the specified username.",
+        },
+    )
+    @action(detail=False, methods=["get"])
+    @validate_request(query_params={"username"})
+    def heatmap(self, request: Request, username: str = None) -> Response:
+        """Get the data to generate a heatmap for the volunteer.
+
+        This includes one entry for every weekday and every hour containing the
+        number of transcriptions made in that time slot.
+        For example, there will be an entry for Sundays at 13:00 UTC, counting
+        how many transcriptions the volunteer made in that time.
+
+        The week days are numbered Sunday=1 through Saturday=7 (blame Django for that).
```

Comment:
Python's normal date scheduling is 0-6, where 0 is a Monday and Sunday is a 6. If you use `isoweekday`, then Monday will be 1 and Sunday will be 7. According to the Django docs that I can find, it looks like it defaults to the normal date scheduling (0-6, Monday-Sunday) unless you use the `ExtractIsoWeekDay` function (1-7, Monday-Sunday). https://docs.djangoproject.com/en/3.2/ref/models/database-functions/#django.db.models.functions.ExtractIsoWeekDay

Reply:
In that case I will use the ISO function, because I don't like Sunday being the first day of the week